src/RecoveryServices/RecoveryServices.Backup.Models/AzureVmModels/AzureVmItem.cs

```diff
@@ -43,6 +43,7 @@ public class AzureVmItem : AzureItem
                 EnumUtils.GetEnum<ItemProtectionState>(protectedItem.ProtectionState.ToString());
             ProtectionStatus = EnumUtils.GetEnum<ItemProtectionStatus>(protectedItem.ProtectionStatus);
             VirtualMachineId = protectedItem.VirtualMachineId;
+            HealthStatus = protectedItem.HealthStatus;
```

Comment:
Please check with Kartik if health status needs to be shown as a part of table output.